shotover-proxy/Cargo.toml

```diff
@@ -53,7 +53,7 @@ halfbrown = "0.1.11"
 # Transform dependencies
 redis-protocol = "3.0.1"
 cassandra-proto = { git = "https://github.com/shotover/cassandra-proto", branch = "move-to-bytes", features = ["v4"] }
-rdkafka = { version = "0.27", features = ["cmake-build"] }
+rdkafka = { version = "0.27" }
```

Comment:
Is there anything else still depending on cmake? Or can we get rid of it completely? (I'll look into it for the next managed service release.) 🤔